src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -801,43 +808,22 @@ export class ServerView extends DirMixin(PolymerElement) {
     }
   }
 
-  _formatBytesTransferred(numBytes, emptyValue = '') {
+  _formatBytesTransferred(numBytes, language, emptyValue = '') {
     if (!numBytes) {
       // numBytes may not be set for manual servers, or may be 0 for
       // unused access keys.
       return emptyValue;
     }
-
-    // Show 0 decimals for < 1MB, 1 decimal for >= 1MB, 2 decimals for >= 1GB.
-    let numDecimals = 0;
-    if (numBytes >= 10 ** 9) {
-      numDecimals = 2;
-    } else if (numBytes >= 10 ** 6) {
-      numDecimals = 1;
-    }
-
-    return byte_size(numBytes, {precision: numDecimals}).toString();
+    return i18n.formatBytes(numBytes, language);
   }
 
-  _getFormattedTransferredUnit(numBytes, emptyValue = '') {
-    const formattedTransfer = this._formatBytesTransferred(numBytes);
-    if (!formattedTransfer) {
-      return emptyValue;
-    }
-    const units = formattedTransfer.match(/[a-zA-Z%]+/g);
-    if (!units || units.length < 0) {
-      return emptyValue;
-    }
-    return units.pop();
-  }
-
-  _getFormattedTransferredValue(numBytes, emptyValue = '') {
-    const formattedTransfer = this._formatBytesTransferred(numBytes);
-    if (!formattedTransfer) {
-      return emptyValue;
+  _formatMonthlyCost(monthlyCost) {
```

Comment:
This should take the language as input, so it's recalculated if the language changes.

Reply:
Done in 9d11943e51379020fda0b9357457e185e844dfc3.  I also added the language dependency to the server location name so that will update as well